### PR TITLE
*: move irrelevant code out of package "distsql"

### PR DESCRIPTION
**Pull request by zz-jason**

### CODE REVIEW

File: plan/plan_to_pb_test.go [hunk 1]
```diff
@@ -131,28 +121,3 @@ func (s *testDistsqlSuite) TestIndexToProto(c *C) {
 	c.Assert(pIdx.IndexId, Equals, int64(1))
 	c.Assert(pIdx.Unique, Equals, true)
 }
-
-type mockResponse struct {
```
Comment (shenli) [inline plan/plan_to_pb_test.go]: This is not used any more?

Comment (zz-jason) [reply plan/plan_to_pb_test.go]: not sure, reverted.

**Comment by coocood** [comment]
LGTM

**Comment by winoros** [comment]
@zz-jason Will you add unit test in `distsql_test.go` later?

**Comment by zz-jason** [comment]
@winoros yes.

**Pull request merged**